.github/workflows/bump_packages.yml

```diff
@@ -0,0 +1,94 @@
+name: Update packages
+on:
+  workflow_dispatch:
+env:
+  GIT_AUTHOR_NAME: ${{ github.actor }}
+  GIT_AUTHOR_EMAIL: ${{ github.actor }}@users.noreply.github.com
```

Comment:
We can argue whether we should set the author to the one who triggered or just set a bot name. I'm leaning to the latter.